test/helpers/utils.js

```diff
@@ -162,6 +176,7 @@ async function getDefaultArgs() {
     factory,
     hegicAdapter,
     mockGammaAdapter,
+    charmAdapter,
     gammaAdapter,
     mockGammaController,
     protocolAdapterLib,
```

Comment:
getting weird CI build errors with [charm adapter](https://github.com/ribbon-finance/structured-products/runs/2162350161?check_suite_focus=true#step:10:41) / [gamma adapter](https://github.com/ribbon-finance/structured-products/runs/2162350161?check_suite_focus=true#step:10:277) that I do not get locally...

Reply:
Those CI errors are mostly from block timing errors, we need to do `hardhat_reset` back to the project's block number when we're done with the RibbonVolatility tests